example_test.go

```diff
@@ -135,6 +135,42 @@ func Example_helpText() {
 	//   --help, -h             display this help and exit
 }
 
+// This example shows the usage string generated by go-arg with custom datanames
+func Example_helpDataname() {
+	// These are the args you would pass in on the command line
+	os.Args = split("./example --help")
+
+	var args struct {
```

Comment:
I'd suggest removing all fields that don't use the new tag from this example. Also, would you add a slide-valued field that uses the new tag?